assets/lib/_variables.scss

```diff
@@ -164,6 +165,9 @@ $related-links-heading: $color-black;
 // Feedback -advice
 $color-call-us-number: $color-green-primary;
 
+// MAS Blog
+$color-mas-blog-blue: $color-blue-cerulean;
```

Comment:
In fact if you just append this with `!default` then apps can override this variable if necessary